src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/w3c/RecordedPerformanceEntries.kt

```diff
@@ -4,7 +4,7 @@ package com.atlassian.performance.tools.jiraactions.api.w3c
  * Holds interesting recorded performance entries.
  * They share the same timeline and can be cross-examined.
  */
-class RecordedPerformanceEntries internal constructor(
+data class RecordedPerformanceEntries internal constructor(
```

Comment:
Could you explain why we want/need the data class here?

Reply:
There were need for hashCode/equals and toString in tests hence `data class` doing what we need.
Plus copy() which we don't need. I might just implement those 3 methods as well